source/index.html.md

```diff
@@ -963,6 +964,23 @@ Examples:
 }
 ````
 
+### Stop recurring status
+
+On any recurring transaction, issuers can respond with <i>stop recurring</i>,
+withdrawing the consent from the cardholder for a merchant to initiate
+transactions on their behalf.
+
+A <i>stop recurring</i> status is both a decline of the current authorization,
+but also a notice to halt all future recurring transactions on this card from
+this merchant.
+
+If <i>stop recurring</i> responses are not adhered to and recurring
+transactions on this card continues, a fee pr. transaction is incurred by
```

Comment:
Should we mention the size of the fee?

Reply:
No. This might change. And very fast gets very complex when you factor in the different scenarios where a fee can be incurred. The fees for Mastercard and Visa are also not the same.

Reply:
Reading this section I was even thinking: why get into scheme politics at all? Shouldn't this documentation stick to protocol and leave the rest to contracts?

Reply:
Fair point... Thinking.